app/services/bulk_upload/sales/log_creator.rb

```diff
@@ -0,0 +1,70 @@
+class BulkUpload::Sales::LogCreator
```

Comment:
maybe for another PR but if we don't want to duplicate this class it may be an idea to have `BulkUpload::LogCreator` where we can inject in the correct `csv_parser`